From e263e19aa1c63dbcbe710e8aae79c8e298606e4c Mon Sep 17 00:00:00 2001
From: Peter Schiffer <pschiffe@redhat.com>
Date: Tue, 4 Nov 2014 14:49:57 +0100
Subject: [PATCH] don't use /usr/bin/env in shebang

There might be an issue that the script is executed with unwanted version of
<lang> if that language is provided by enabled dynamic software collection.

Resolves: #987069
---
 contrib/chem/chem.pl       | 2 +-
 contrib/groffer/groffer.pl | 2 +-
 contrib/groffer/roff2.pl   | 2 +-
 src/roff/grog/grog.pl      | 2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

Index: groff-1.22.4/contrib/chem/chem.pl
===================================================================
--- groff-1.22.4.orig/contrib/chem/chem.pl
+++ groff-1.22.4/contrib/chem/chem.pl
@@ -1,4 +1,4 @@
-#! /usr/bin/env perl
+#! /usr/bin/perl
 
 # chem - a groff preprocessor for producing chemical structure diagrams
 
Index: groff-1.22.4/contrib/groffer/groffer.pl
===================================================================
--- groff-1.22.4.orig/contrib/groffer/groffer.pl
+++ groff-1.22.4/contrib/groffer/groffer.pl
@@ -1,4 +1,4 @@
-#! /usr/bin/env perl
+#! /usr/bin/perl
 
 # groffer - display groff files
 
Index: groff-1.22.4/contrib/groffer/roff2.pl
===================================================================
--- groff-1.22.4.orig/contrib/groffer/roff2.pl
+++ groff-1.22.4/contrib/groffer/roff2.pl
@@ -1,4 +1,4 @@
-#! /usr/bin/env perl
+#! /usr/bin/perl
 
 # roff2* - transform roff files into other formats
 
Index: groff-1.22.4/src/roff/grog/grog.pl
===================================================================
--- groff-1.22.4.orig/src/roff/grog/grog.pl
+++ groff-1.22.4/src/roff/grog/grog.pl
@@ -1,4 +1,4 @@
-#! /usr/bin/env perl
+#! /usr/bin/perl
 # grog - guess options for groff command
 # Inspired by doctype script in Kernighan & Pike, Unix Programming
 # Environment, pp 306-8.
Index: groff-1.22.4/contrib/glilypond/glilypond.pl
===================================================================
--- groff-1.22.4.orig/contrib/glilypond/glilypond.pl
+++ groff-1.22.4/contrib/glilypond/glilypond.pl
@@ -1,4 +1,4 @@
-#! /usr/bin/env perl
+#! /usr/bin/perl
 
 package main;
 
Index: groff-1.22.4/contrib/gperl/gperl.pl
===================================================================
--- groff-1.22.4.orig/contrib/gperl/gperl.pl
+++ groff-1.22.4/contrib/gperl/gperl.pl
@@ -1,4 +1,4 @@
-#! /usr/bin/env perl
+#! /usr/bin/perl
 
 # gperl - add Perl part to groff files, this is the preprocessor for that
 
Index: groff-1.22.4/contrib/gpinyin/gpinyin.pl
===================================================================
--- groff-1.22.4.orig/contrib/gpinyin/gpinyin.pl
+++ groff-1.22.4/contrib/gpinyin/gpinyin.pl
@@ -1,4 +1,4 @@
-#! /usr/bin/env perl
+#! /usr/bin/perl
 
 # gpinyin - European-like Chinese writing 'pinyin' into 'groff'
 
Index: groff-1.22.4/contrib/gpinyin/subs.pl
===================================================================
--- groff-1.22.4.orig/contrib/gpinyin/subs.pl
+++ groff-1.22.4/contrib/gpinyin/subs.pl
@@ -1,4 +1,4 @@
-#! /usr/bin/env perl
+#! /usr/bin/perl
 
 # gpinyin - European-like Chinese writing 'pinyin' into 'groff'
 
Index: groff-1.22.4/contrib/groffer/main_subs.pl
===================================================================
--- groff-1.22.4.orig/contrib/groffer/main_subs.pl
+++ groff-1.22.4/contrib/groffer/main_subs.pl
@@ -1,4 +1,4 @@
-#! /usr/bin/env perl
+#! /usr/bin/perl
 
 # groffer - display groff files
 
Index: groff-1.22.4/contrib/groffer/man.pl
===================================================================
--- groff-1.22.4.orig/contrib/groffer/man.pl
+++ groff-1.22.4/contrib/groffer/man.pl
@@ -1,4 +1,4 @@
-#! /usr/bin/env perl
+#! /usr/bin/perl
 
 # groffer - display groff files
 
Index: groff-1.22.4/contrib/groffer/subs.pl
===================================================================
--- groff-1.22.4.orig/contrib/groffer/subs.pl
+++ groff-1.22.4/contrib/groffer/subs.pl
@@ -1,4 +1,4 @@
-#! /usr/bin/env perl
+#! /usr/bin/perl
 
 # groffer - display groff files
 
Index: groff-1.22.4/mdate.pl
===================================================================
--- groff-1.22.4.orig/mdate.pl
+++ groff-1.22.4/mdate.pl
@@ -1,4 +1,4 @@
-#! /usr/bin/env perl
+#! /usr/bin/perl
 #
 # Copyright (C) 1991-2018 Free Software Foundation, Inc.
 # 
Index: groff-1.22.4/src/roff/grog/subs.pl
===================================================================
--- groff-1.22.4.orig/src/roff/grog/subs.pl
+++ groff-1.22.4/src/roff/grog/subs.pl
@@ -1,4 +1,4 @@
-#! /usr/bin/env perl
+#! /usr/bin/perl
 # grog - guess options for groff command
 # Inspired by doctype script in Kernighan & Pike, Unix Programming
 # Environment, pp 306-8.
Index: groff-1.22.4/tmac/hyphenex.pl
===================================================================
--- groff-1.22.4.orig/tmac/hyphenex.pl
+++ groff-1.22.4/tmac/hyphenex.pl
@@ -1,4 +1,4 @@
-#! /usr/bin/env perl
+#! /usr/bin/perl
 #
 #
 # hyphenex.pl
